test/db.tests.js

```diff
@@ -1021,6 +1021,64 @@ describe('LimitDBRedis', () => {
           done();
         });
       })
+
+      describe('overrides', () => {
+        it('should use elevated_limits config override when provided', (done) => {
```

Comment:
We also need `should use root level erl config if not provided with erl config override` or something alike

Reply:
isn't that what all the previous tests are testing?

Reply:
Ah I see what you mean. Providing an overriding without elevated_limits, is it that?

Reply:
> Ah I see what you mean. Providing an override without elevated_limits, is it that?

Yes! But I think we have a test for it already (i.e. standard override tests + this test). After what you said about overrides in DM (i.e. they can't be partial), guess we will have to skip this test case